pkgs/by-name/no/nom/package.nix

```diff
@@ -5,13 +5,13 @@
 }:
 buildGoModule rec {
   pname = "nom";
-  version = "2.7.2";
+  version = "2.7.3";
 
   src = fetchFromGitHub {
     owner = "guyfedwards";
     repo = "nom";
     rev = "v${version}";
```

Comment:
rev -> tag

Reply:
The reason is that you don't want to accidentally fetch a branch with the same name.